build-dotty.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eux
+set -o pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >& /dev/null && pwd)"
+BUILD_DIR="$(mktemp -d)"
+BUILD_LOG="$ROOT/build_log"
+VERSION_FILE="$ROOT/dotty_version"
+REFERENCE=""
+
+REFERENCE=${DOTTY_REFERENCE:-}
+
+if [ -z $REFERENCE ]; then
+    rm -f "$VERSION_FILE"
+    exit 0
+fi
+
+
+pushd "$BUILD_DIR"
+git clone --recursive https://github.com/lampepfl/dotty.git . --depth=30
```

Comment:
Not sure it is a good idea to limit the depth. You won't find the reference commit if is to deep

Reply:
Remove `--recursive`